service/dap/daptest/client.go

```diff
@@ -49,30 +49,96 @@ func (c *Client) send(request dap.Message) {
 	dap.WriteProtocolMessage(c.conn, request)
 }
 
-// ReadBaseMessage reads and returns a json-encoded DAP message.
-func (c *Client) ReadBaseMessage() ([]byte, error) {
-	message, err := dap.ReadBaseMessage(c.reader)
+func (c *Client) ExpectDisconnectResponse(t *testing.T) *dap.DisconnectResponse {
```

Comment:
Should these be calling t.Helper()?
(Just learned about it from PR #1882)

Reply:
Done